server/reload.go

```diff
@@ -735,8 +728,12 @@ func (s *Server) diffOptions(newOpts *Options) ([]option, error) {
 			diffOpts = append(diffOpts, &clientAdvertiseOption{newValue: cliAdv})
 		case "accounts":
 			diffOpts = append(diffOpts, &accountsOption{})
-		case "trustedkeys":
-			diffOpts = append(diffOpts, &trustKeysOption{})
+		case "accountresolver":
+			// We can't move from no resolver to one. So check for that.
+			if oldValue == nil && newValue != nil {
```

Comment:
and do we support removing?

Reply:
Not really. I could update the conditional.

Reply:
Should be fixed now.